config/development.json

```diff
@@ -175,7 +175,7 @@
 		"signup/wpcc": true,
 		"signup/wpforteams": true,
 		"site-indicator": true,
-		"site-level-billing": true,
+		"site-level-billing": false,
```

Comment:
We can actually remove this flag entirely from this file and `config/horizon` because it's no longer used anywhere after this PR.